epinorm/geo.py

```diff
@@ -30,9 +31,17 @@
 
 
 class Geocoder:
+    
+    TIME_LAST_REQUEST = time()
```

Comment:
As far as I know, the python convention is to use UPPER_CASE_NAMES only for naming global constants. Here we are dealing with a regular class attribute, so it should be in lower case I think.

```suggestion
    time_last_request = time()
```